Doc/library/unittest.mock.rst

```diff
@@ -957,7 +959,7 @@ object::
         >>> mock.method.assert_awaited_once()
         Traceback (most recent call last):
         ...
-        AssertionError: Expected mock to have been awaited once. Awaited 2 times.
+        AssertionError: Expected method to have been awaited once. Awaited 0 times.
```

Comment:
This is likely not the intended output, if called with `mock.assert_awaited_once()` rather than `mock.method.assert_awaited_once()` it yields the output that was documented. Should I change it immediately?

Reply:
Fix it here :-)

Reply:
Awesome! Done [d7f3b13](https://github.com/python/cpython/pull/109353/commits/d7f3b135cfe21bfda530901877188b8a623a6429) ✅